tests/Unit/Controllers/Store/CentreControllerTest.php

```diff
@@ -213,7 +214,8 @@ public function testItCanDownloadASocialPrescriptionsRegistrationsSpreadsheet()
             "Leaving Date",
             "Leaving Reason",
             "Rejoin Date",
-            "Leave Count"
+            "Leave Count",
+            "Days on programme"
         ];
         // The unexpected headers
         $unexpected_headers = [
```

Comment:
Are we not testing the actual functionality?

Reply:
I've added to the existing card about fixing tests: https://trello.com/c/VCK9yl9L/1958-improve-fix-skipped-tests